ansible/prepare_host_env.sh

```diff
@@ -0,0 +1,36 @@
+if [ "$EUID" -ne 0 ]
```

Comment:
Missing shellbang; This file isn't used anywhere

Reply:
It is used manually. I'll create readme for that.